src/render/shade.rs

```diff
@@ -136,15 +140,18 @@ pub struct ParamDictionary {
     pub textures: Vec<NamedCell<TextureParam>>,
 }
 
-/// An associated link structure for `ParamDictionary` that redirects program
-/// input to the relevant dictionary cell.
+/// An associated parameter link for ParamDictionary
```

Comment:
Could we change this comment to "Redirects program input to the relevant `ParamDictionary` cell"